config/config.yaml

```diff
@@ -18,6 +22,8 @@ reference_gff3: resources/reference/Anopheles-gambiae-PEST_BASEFEATURES_AgamP4.1
 # .bed file of target SNPs
 bed: "resources/AgamDao_wholegenome_snptargets.bed"
 
+BCL_conversion:
```

Comment:
I wonder whether this is necessary? If users already have reads, they just put them in the `resources/reads/` folder, otherwise, AmpSeeker expects an Illumina_dir to run BCL conversion on?

Reply:
I'm not sure whether the pipeline is robust enough to carry on execution with fastqs available. That's why I included that config switch to make it easy to switch off bcl_convert instead of commenting it out or cutting it out at run time. I'm open to suggestions for another way to do this.

Reply:
It should do. In fact, we can explicitly test with the test data, because we start from the reads...

Reply:
Im going to remove this line and see if the CI works.

Reply:
Ohh yes that makes sense. So would that mean bcl_convert.smk would be invoked automatically if the workflow does not detect `resources/reads` at first?

Reply:
exactly :) snakemake will know that we need fastqs in resources/reads/, but if they are already there, it will assume we will already have done the bcl-convert step :) 

If they arent present, it knows it needs to run bcl-convert.

Reply:
The changes will already be committed now, I did so in the PR automatically (sorry should have said). 

If you could just remove some of the commented out lines in bcl-convert.smk, then we'll be ready to merge :)